docs/getting-started/your-first-mock.md

```
@@ -0,0 +1,384 @@
+# Using Docker
+docker run -it -p 8000:8000 -v $(pwd)/moclojer.yml:/app/moclojer.yml ghcr.io/moclojer/moclojer:latest
+
+# Using JAR file
+java -jar moclojer.jar --config moclojer.yml
+
+# Using native binary
+./moclojer --config moclojer.yml
+```
+
+You should see output similar to:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)